src/main/js/page_objects/blueocean/bluePipelineBranch.js

```diff
@@ -0,0 +1,30 @@
+var url = require('../../util/url');
+
+module.exports = {
+    elements: {
+        runButton: 'div.actions div.run-pipeline',
+        toasts: 'div.toaster div.toast span.text'
```

Comment:
You are selecting the toast test here specifically, right?  Might be better to give it an explicit name in that case e.g. "toastText" ?